recipes/addons.rb

```diff
@@ -70,6 +70,34 @@
 include_recipe "kube-hops::hops-system"
 include_recipe "kube-hops::filebeat"
 
+# Apply node taints
+node['kube-hops']['taints'].split(")").each do |node_taint|
+  node_taint = node_taint[1, node_taint.length-1]
+  node_name = node_taint.split(",")[0]
+  taint = node_taint.split(",")[1]
+
+  kube_hops_kubectl "#{taint}" do
+    user node['kube-hops']['user']
+    group node['kube-hops']['group']
+    k8s_node node_name
+    action :taint
+  end
+end
+
+# Apply node labels
+node['kube-hops']['labels'].split(")").each do |node_label|
+  node_label = node_label[1, node_label.length-1]
+  node_name = node_label.split(",")[0]
+  label = node_label.split(",")[1]
```

Comment:
Just by curiosity, why not store the split in a variable?

Reply:
cleaned up.